cirq-core/cirq/testing/consistent_decomposition.py

```diff
@@ -47,3 +47,20 @@ def assert_decompose_is_consistent_with_unitary(val: Any, ignoring_global_phase:
     else:
         # coverage: ignore
         np.testing.assert_allclose(actual, expected, atol=1e-8)
+
+
+def assert_decompose_ends_at_default_gateset(val: Any):
+    """Ensures that all cirq gate decompositions end at the default cirq gateset."""
+
+    # pylint: disable=unused-variable
+    __tracebackhide__ = True
```

Comment:
What is this for ?

Reply:
This is useful to hide the traceback of assertion calls, in order to avoid clutter in user code if tests fail. 

It's used as a general pattern across all cirq/testing/ methods. I've removed it for now as I think the assertion trace is very useful here, especially while this is still a WIP and further decompositions are being added in subsequent PRs.